tests/test_add_mds_to_cluster.py

```diff
@@ -49,8 +49,6 @@ def setup(self):
     CEPH_OBJ = OCS(**self.fs_data)
```

Comment:
@RazTamir 
Why all are using OCS() class instead of StorageClass() ?
I still see StorageClass(OCS) inheriting and my master branch is uptodate .

Any other guidelines for the usage ?

Reply:
things are changing every day :)
so we realized most of the functionality is around get(), create(), apply(), etc. so why do we need to maintain all those classes?
for these helper functions, it does make sense to have them inside a PVC class.
So, in general, I'd say it depend on the needs.
For example, for storage class (I'm still not familiar with all actions) I'm not sure we need more functionality other than create() and delete() so for now not in use so people are using OCS directly.

Apologies for all the changes but I hope we reached some good point where we are on the right direction

Reply:
In last PR https://github.com/red-hat-storage/ocs-ci/pull/132 we agreed to defer multiple classes to keep things simple because there wasn't functionality specific to class that needed to be implemented, If the *future* has reached today then there should be one more discussion :)

Reply:
I think the confusion is we still have to StorageKind and PVC class, probably should get rid of those in the interim.

Reply:
@RazTamir @vasukulkarni 
Yes true, I got confused because of those files and I mislead Shreekara as well.

Reply:
> things are changing every day :)
> so we realized most of the functionality is around get(), create(), apply(), etc. so why do we need to maintain all those classes?
As you could already see these are just basic functions required but in a larger context , for ex: In PVC resize you would use apply as one of the step but apply() itself doesn't have significance in carrying semantics of the actions on particular kind.
This is the pattern we were discussing about where there is need of classes for each kind and grouping the methods.

Anyways as we started with simplified approach, going forward we can expand on need basis.

> for these helper functions, it does make sense to have them inside a PVC class.
> So, in general, I'd say it depend on the needs.
> For example, for storage class (I'm still not familiar with all actions) I'm not sure we need more functionality other than create() and delete() so for now not in use so people are using OCS directly.
> 
> Apologies for all the changes but I hope we reached some good point where we are on the right direction